infra/modular-examples/gcp-google-workspace/main.tf

```diff
@@ -340,3 +340,15 @@ output "todos_3" {
     values(module.worklytics-psoxy-connection)[*].todo
   )
 }
+
+
+# use case: let someone consume this deploy another psoxy instance, reusing artifacts
```

Comment:
:+1: 
Probably something we could add by default: compile once and upload to the bucket in both platforms (AWS, GCP)